tests/python/relax/frontend/test_onnx_frontend.py

```diff
@@ -0,0 +1,1378 @@
+        ]
+        graph_outputs = [helper.make_tensor_value_info("output", TensorProto.FLOAT, output_shape)]
+
+        pad_node = helper.make_node(
+            "Pad", ["input", "pads", "constant_value"], ["output"], mode="constant"
+        )
+        graph_nodes = [pad_node]
+        graph = helper.make_graph(
+            graph_nodes,
+            "test_pad_constant_value",
+            inputs=graph_inputs,
+            outputs=graph_outputs,
+            initializer=initializer,
+        )
+        model = helper.make_model(
+            graph,
+            producer_name="test_pad_constant_value",
+        )
+        check_correctness(model)
+
+    verify_pad_constant_value(0)
+
+
+@pytest.mark.parametrize("fp_arith", [np.float16, np.float32])
+@pytest.mark.parametrize("dynamic", [True, False])
+def test_split(fp_arith, dynamic):
+    """test_split"""
+
+    def verify_split(indata_shape, outdata_shapes, split, axis=0, pass_split=True, opset=11):
+        indata = np.random.normal(size=indata_shape).astype(fp_arith)
+        input_names = ["input"]
+        initializer = []
+
+        if split:
+            split_index = range(len(split))
+        else:
+            split_index = range(len(outdata_shapes))
+
+        indata_shape = list(indata.shape)
+        if dynamic:
+            indata_shape = ["?" for _ in range(len(indata.shape))]
+            outdata_shapes = [["?" for _ in range(len(o))] for o in outdata_shapes]
+
+        inputs = [
+            helper.make_tensor_value_info(
+                "input", mapping.NP_TYPE_TO_TENSOR_TYPE[indata.dtype], indata_shape
+            )
+        ]
+
+        if pass_split:
+            if opset >= 13:
+                np_split = np.array(split).astype(np.int64)
+                initializer.append(
+                    helper.make_tensor("split", TensorProto.INT64, list(np_split.shape), np_split)
+                )
+        node = helper.make_node(
+            "Split",
+            inputs=input_names,
+            outputs=[f"output_{i}" for i in range(len(split_index))],
+            axis=axis,
+        )
+
+        if pass_split and opset < 13:
+            split_attr = helper.make_attribute("split", split)
+            node.attribute.append(split_attr)
+
+        graph = helper.make_graph(
+            [node],
+            "split_test",
+            inputs=inputs,
+            initializer=initializer,
+            outputs=[
+                helper.make_tensor_value_info(
+                    f"output_{i}",
+                    mapping.NP_TYPE_TO_TENSOR_TYPE[indata.dtype],
+                    list(outdata_shapes[i]),
+                )
+                for i in range(len(split_index))
+            ],
+        )
+        model = helper.make_model(graph, producer_name="split_test")
+        check_correctness(model, inputs={"input": indata}, opset=opset)
+
+    # 1D
+    verify_split(6, [[2], [2], [2]], [2, 2, 2], 0)
+    verify_split(6, [[2], [2], [2]], [2, 2, 2], 0, False)
```

Comment:
Probably preferable to indicate the argument names, since some of these are optional